portal-api/src/utils/string.js

```diff
@@ -7,8 +7,16 @@ const isEmptyString = (str) => {
   return false;
 };
 
+/**
+ * Check if a param has a value, i.e
+ * - the param is defined
+ * - the param is not an empty string
+ * - the param is not a 'Select value' string
+ * @param {unknown}
+ * @returns Boolean
+ */
 const hasValue = (str) => {
-  if (str && !isEmptyString(str)) {
+  if (str && !isEmptyString(str) && str !== 'Select value') {
```

Comment:
This is required, otherwise various validation thinks that "Select value" is a valid string.

Reply:
I do not think CSS is a failsafe approach

Unfortunately, some old portal pages have the validation in `portal-api`, rather than the front end.

I don't want to introduce new patterns here, I think it's better to tweak/update the existing validation - wherever it is.

Have updated this to be one line

Reply:
Sorry I also meant with F.E (but server side) validation of 'Select value' as well -- to avoid people posting with it...

I think we might be introducing a new pattern here regardless it seems? `hasValue` is an api function that is coupled with our front end copy -- which feels slightly odd?

Reply:
having this without return true/false would be nicer but if it is not the current pattern, then fine to leave it as is